rest_framework/routers.py

```diff
@@ -124,8 +125,28 @@ class SimpleRouter(BaseRouter):
         ),
     ]
 
-    def __init__(self, trailing_slash=True):
+    def __init__(self, trailing_slash=True, use_regex_path=True):
         self.trailing_slash = '/' if trailing_slash else ''
+        if use_regex_path:
+            self._base_regex = '(?P<{lookup_prefix}{lookup_url_kwarg}>{lookup_value})'
+            self._default_regex = '[^/.]+'
+            self._url_conf = url
+        else:
+            self._base_regex = '<{lookup_value}:{lookup_prefix}{lookup_url_kwarg}>'
+            self._default_regex = 'path'
```

Comment:
I just realized that `path` is not the right converted to use as default, the right one is `str`.
> **str** - Matches any non-empty string, excluding the path separator, `'/'`. This is the default if a converter isn’t included in the expression.
> **path** - Matches any non-empty string, including the path separator, `'/'`. This allows you to match against a complete URL path rather than just a segment of a URL path as with `str`.



I will correct this.